test/index.test.ts

```diff
@@ -19,8 +20,32 @@ describe("Testing Toucan-SDK", function () {
   let addr2: SignerWithAddress;
   let toucan: ToucanClient;
   let swapper: Contract;
-  let scoredTCO2sBCT: string[];
-  let scoredTCO2sNCT: string[];
+
+  /**
+   *
+   * @param poolSymbol Pool symbol to get scored TCO2s for
+   * @returns an array of addresses representing TCO2s in the given pool that have a balance > 1.0
```

Comment:
why greater than 1.0 and not 0?

Reply:
Redeems and retirements of (0,1] are allowed by the protocol and we should not disallow them here.